products/identityplatform/api.yaml

```
@@ -0,0 +1,408 @@
+              URL to send Authentication request to.
+          - !ruby/object:Api::Type::Boolean
+            name: 'signRequest'
+            description: |
+              Indicates if outbounding SAMLRequest should be signed.
+          - !ruby/object:Api::Type::Array
+            name: 'idpCertificates'
+            required: true
+            description: |
+              The IDP's certificate data to verify the signature in the SAMLResponse issued by the IDP.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+              - !ruby/object:Api::Type::String
+                name: 'x509Certificate'
+                description: |
+                  The x509 certificate
+      - !ruby/object:Api::Type::NestedObject
+        name: 'spConfig'
+        required: true
+        description: |
+          SAML SP (Service Provider) configuration when the project acts as the relying party to receive
+          and accept an authentication assertion issued by a SAML identity provider.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'spEntityId'
+            description: |
+              Unique identifier for all SAML entities.
+          - !ruby/object:Api::Type::String
+            name: 'callbackUri'
+            description: |
+              Callback URI where responses from IDP are handled. Must start with `https://`.
+          - !ruby/object:Api::Type::Array
+            name: 'spCertificates'
+            output: true
+            description: |
+              The IDP's certificate data to verify the signature in the SAMLResponse issued by the IDP.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+              - !ruby/object:Api::Type::String
+                name: 'x509Certificate'
+                description: |
+                  The x509 certificate
+  - !ruby/object:Api::Resource
+    name: 'OauthIdpConfig'
+    base_url: 'projects/{{project}}/oauthIdpConfigs'
+    self_link: 'projects/{{project}}/oauthIdpConfigs/{{name}}'
+    create_url: 'projects/{{project}}/oauthIdpConfigs?oauthIdpConfigId={{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      OIDC IdP configuration for a Identity Toolkit project.
+
+      You must enable the
+      ((Google Identity Platform)[https://console.cloud.google.com/marketplace/details/google-cloud-platform/customer-identity])
+      in the marketplace prior to using this resource.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The name of the OauthIdpConfig. Must start with `oidc.`.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        description: |
+          Human friendly display name.
+      - !ruby/object:Api::Type::Boolean
+        name: 'enabled'
+        description: |
+          If allows users to sign in with the provider.
```

nit: this is kind of awkward phrasing
```suggestion
          If this config allows users to sign in with the provider.
```